tools/TestFx/Utilities/KustoUtility.psm1

```diff
@@ -24,7 +46,7 @@ function InitializeKustoPackages {
     New-Item -Path . -Name $kustoPackagesDirectoryName -ItemType Directory -Force
 
     $kustoPackages = @(
-        @{ PackageName = "Azure.Core"; PackageVersion = "1.25.0"; DllName = "Azure.Core.dll" },
+        @{ PackageName = "Azure.Core"; PackageVersion = "1.30.0"; DllName = "Azure.Core.dll" },
```

Comment:
I believe we are using `1.31.0` now.

Reply:
This v1.30.0 is a dependency for Kusto and Email communication service as a separate package. No need to be consistent with the version we use in Az.